Corona-Warn-App/src/main/res/layout/include_submission_contact.xml

```diff
@@ -109,6 +109,8 @@
                         android:layout_height="wrap_content"
                         android:layout_marginTop="@dimen/spacing_small"
                         android:text="@string/submission_contact_body_other"
+                        android:autoLink="phone"
+                        android:textColorLink="@color/colorTextTint"
```

Comment:
Thanks for this. I think it does not work for All Android versions.
You fix worked on Pixel 4a - Android 11. 
I have tested it on S8 Android 9 and emulator Galaxy Nexus Android 6 and the fix is not working on these versions.
We will have to do programatically in this case.
 maybe an extension function to be used in other places too:
```kotlin
fun TextView.linkifyPhoneNumbers(){
    LinkifyCompat.addLinks(
        this,
        Patterns.PHONE,
        "tel:",
        Linkify.sPhoneNumberMatchFilter,
        Linkify.sPhoneNumberTransformFilter
    )
    movementMethod = LinkMovementMethod.getInstance()
    setLinkTextColor(context.getColorCompat(R.color.colorTextTint))
}
```

Reply:
@mtwalli 
Thanks for your feedback and it confirms what I found in https://stackoverflow.com/questions/40788608/androidautolink-for-phone-numbers-doesnt-always-work which shows how the effect of autoLink depends on the device's locale.

In a Nexus 6 emulator, Android 6 (API 23) with the device language set to English (United States) the telephone number 116 117 in the App Information > Technical Hotline view (which this PR does not change) is **not** linked. I did also a spot check on a Samsung Galaxy A5 (2017), Android 8 when the locale is English (United States) the telephone number 116 117 is not linked. It does appear linked when the language is changed to English (United Kingdom). 

Using a Pixel 3a emulator, Android 11 I got a different result: the telephone number 116 117 was never shown linked, independently of the locale. I saw in a short test program that Android 11 only ever linked 7-digit and 10-digit telephone numbers and international phone numbers like +49 xx in a TextView.

So the conclusion is that autoLink for phone numbers is not reliable.

The changes I proposed in this PR simply align the Technical Hotline view with the Request TAN view.

If the phone number 116 117 in the Technical Hotline view is shown linked in a certain environment, then the phone number 116 117 in the Request TAN view is also shown linked. If the link does not work in the Technical Hotline view, it will consistently not be working in the Request TAN view either.

The issue is much more complex than I imagined and a better implementation may need to be deferred. LinkifyCompat looks like it would provide a general solution. For the single phone number "Medical Emergency Service Hotline" 116 117 it may be simpler to make it a special case and implement a calling button or similar.

In summary, the changes proposed in this PR improve the situation for the Request TAN view in some environments and don't make it any worse. The changes are however not sufficient to make the phone number 116 117 clickable in all environments.

Reply:
@MikeMcC399 Thank you for the detailed information.
Based on our discussion on https://github.com/corona-warn-app/cwa-app-android/pull/2286
@PhilippNowak96  kindly implemented the proposed solution there, if you confirm that the changes in https://github.com/corona-warn-app/cwa-app-android/pull/2286 fixes it on the devices you mentioned above, can we close this PR in favour of https://github.com/corona-warn-app/cwa-app-android/pull/2286?

Reply:
I agree that this PR can be closed in favour of #2286.